platform/view/services/storage/driver/sql/common/binding.go

```diff
@@ -94,6 +94,9 @@ func (db *BindingStore) PutBindings(ctx context.Context, longTerm view.Identity,
 	if len(ephemerals) == 0 {
 		return nil
 	}
+	if len(ephemerals) > 1000 {
+		return errors.Errorf("Too many ephemerals (%d). Max allowed is 1000", len(ephemerals))
+	}
```

Comment:
Please update the go docs for this function to clearly describe that here is a limit.